packages/scandipwa/src/component/CategorySearch/CategorySearch.component.js

```diff
@@ -1,68 +0,0 @@
-/**
- * ScandiPWA - Progressive Web App for Magento
- *
- * Copyright © Scandiweb, Inc. All rights reserved.
- * See LICENSE for license details.
- *
- * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
- * @package scandipwa/base-theme
- * @link https://github.com/scandipwa/base-theme
- */
```

Comment:
We don't use this component anywhere ?

Reply:
We don't :)
During previous refactoring related to prop-types I found 6 more not used components and also had to delete them.

Reply:
Seems deprecated. Interesting if we have even more of those :face_exhaling:. Anyway cool that you deleted them.